app/workers/publishing_api_corporate_information_pages_worker.rb

```diff
@@ -0,0 +1,39 @@
+class PublishingApiCorporateInformationPagesWorker
+  extend Forwardable
+  include Sidekiq::Worker
+
+  def perform(edition_id, event)
```

Comment:
There are events that we don't care about such as `schedule`, `unschedule`, `force_schedule`.